testsuite/forge.py

```diff
@@ -1118,28 +1249,52 @@ def list_jobs(
 ) -> None:
     """List all available clusters"""
     shell = LocalShell()
-    old_cluster = get_current_cluster_name(shell)
+    filesystem = LocalFilesystem()
+    processes = SystemProcesses()
+    context = SystemContext(shell, filesystem, processes)
+
+    # Default to show running jobs
+    phase = phase or ["Running"]
+
     pattern = re.compile(regex or ".*")
-    try:
-        for cluster in list_eks_clusters(shell):
-            set_current_cluster(shell, cluster)
-            print("Cluster:", cluster)
-            for job in get_forge_jobs(shell):
-                if not pattern.match(job.name) or phase and job.phase not in phase:
-                    continue
-                if job.succeeded():
-                    fg = "green"
-                elif job.failed():
-                    fg = "red"
-                elif job.running():
-                    fg = "yellow"
-                else:
-                    fg = "white"
+    jobs = asyncio.run(get_all_forge_jobs(context))
+
+    for job in jobs:
+        if not pattern.match(job.name) or phase and job.phase not in phase:
+            continue
+        if job.succeeded():
+            fg = "green"
+        elif job.failed():
+            fg = "red"
+        elif job.running():
+            fg = "yellow"
+        else:
+            fg = "white"
 
-                click.secho(f"{job.name} {job.phase}", fg=fg)
-    except Exception as e:
-        set_current_cluster(shell, old_cluster)
-        raise e
+        click.secho(f"{job.cluster.name} {job.name} {job.phase}", fg=fg)
+
+
+@main.command()
+@click.argument("job_name")
+def tail(
```

Comment:
where is this used?

Reply:
This is a command which you and i can run